documentation/book/con-certificate-renewal.adoc

```diff
@@ -0,0 +1,31 @@
+[id='con-certificate-renewal-{context}']
+= Certificate Renewal
+
+CA certificates (_cluster CA_ and _clients CA_) are only valid for given number of days after they are generated.
+When using auto-generated CA certificates this time period defaults to 365 days.
+Manually installed CA certificates will come with their own validity period.
+
+If a CA certificate has expired then certificates signed by it will not be valid, even if they are still within their own period of validity. This usually means that new client certificates must also be generated when a CA certificate is going to expire.
+
+While CA certificates are being replaced it is necessary for peers to trust certificates signed by either the old or the new CA. 
+This ensures continued operation of the cluster.
+
+The process for CA certificate renewal is:
```

Comment:
We should break this out into a procedure.

Reply:
Yeah, eventually, but it's actually more complex than this because it breaks into a couple of cases depending on whether the user is using auto-generated CAs, and whether it's the cluster or clients CA. I don't want to document this now as I'm still coding it. So I would prefer to leave this to another PR.

Reply:
Okey dokey! Let's make sure we log an issue